modules/serverless-installing-cli-linux.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_{context}"]
+= Installing `kn` for Linux
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active OpenShift Enterprise subscription on your Red Hat account:
+
+[source]
+----
+# subscription-manager register
+# subscription-manager refresh
+# subscription-manager attach --pool=<pool_id> <1>
+# subscription-manager repos --enable="openshift-serverless-1-for-rhel-8-x86_64-rpms"
+# yum install atomic-openshift-clients
+----
+<1> Pool ID for an active OpenShift Enterprise subscription
+
+For Linux distributions, you can download the CLI directly from the link:TBD[Red Hat Customer Portal] as a *_tar.gz_* archive. 
```

Comment:
* Don't forget to replace the TBD with the correct link. For the customer portal generically, the URL is https://access.redhat.com/.
* For file names (tar.gz), we use backticks only. Check other instances in the PR too.